Explorer/Assets/DCL/StylizedSkybox/Scripts/SkyboxController.cs

```diff
@@ -1,13 +1,11 @@
 using DCL.Diagnostics;
 using DCL.FeatureFlags;
 using DCL.StylizedSkybox.Scripts;
-using System;
 using UnityEngine;
 using UnityEngine.Rendering;
 
 public class SkyboxController : MonoBehaviour
 {
-    private const int SECONDS_IN_DAY = 86400;
```

Comment:
Currently the PR is not compiling because you are missing one `SECONDS_IN_DAY` change to use the settings asset.